scripts/lib/LinkChecker.test.ts

```diff
@@ -74,7 +74,7 @@ describe("Validate links", () => {
     let testFile = new File("docs/testpath.mdx", [], false);
     const results = await testLink.checkLink([testFile]);
     expect(results).toEqual([
-      "❌ /testorigin.mdx: Could not find link '/test-alternative-path'",
+      "❌ /testorigin.mdx: Could not find link '/test-alternative-path'.",
```

Comment:
Nit: I think it's better to not have the period. It's kinda noisy and makes copying and pasting more annoying

Reply:
Done!